berkeleygw/si-bands/run-all.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+set -e
+
+export OMP_NUM_THREADS=1
+export CMD_PREFIX_QE="mpirun --allow-run-as-root -np 4 apptainer run /root/qe-7.4-gcc-openmpi-openblas.sif"
+export CMD_PREFIX_BGW="mpirun --allow-run-as-root -np 4 apptainer run /root/berkeleygw-4.0-gnu-almalinux9.sif"
```

Comment:
If the jobs are submitted via a job manager (e.g., pbs), we need to implement a wait for the current job to complete before running the next step. See https://github.com/Exabyte-io/cli-job-examples/blob/2b8fc516114f9cc97b0f061b6b19d1852324f4dc/deepmd/train_mlff_qe-cp-traj/run.sh#L35